design/ui/pagination/README.mdx

```diff
@@ -0,0 +1,170 @@
+      padded
+      // eslint-disable-next-line react/jsx-no-bind
+      renderBodyRow={renderBodyRow} // ^^ bending the law!
+      tableData={data}
+    />
+  );
+};
+ExternalTable.propTypes = {
+  data: PropTypes.arrayOf(PropTypes.shape({
+    id    : PropTypes.number,
+    email : PropTypes.string,
+  })).isRequired,
+};
+
+/**
+ * Since the behaviour of our commponent depends on its input, we must create an
+ * environment capable of interacting with it.
+ *
+ * The data and its presentation are handled by this component (the external world).
+ * Our test subject (Pagination) is only responsible for receiving the current values
+ * for page and limit and notify (the external world) about any changes generated by
+ * the means of children components under its domain.
+ *
+ * The state of this component tracks page and limit as well, this is because we are
+ * pretending to be connected to an external data source and we are passing the response. */
+export class Wrapper extends PureComponent {
+
+  static propTypes = {
+    entityNamePlural   : PropTypes.string,
+    entityNameSingular : PropTypes.string,
+    totalItems         : PropTypes.number,
+  };
+
+  static defaultProps = {
+    entityNamePlural   : 'Total',
+    entityNameSingular : 'Total',
+    totalItems         : dataFixture.length,
+  };
+
+  state = {
+    data    : [].concat(dataFixture).slice(0, 10),
+    limit   : 10,
+    page    : 1,
+    rawData : [].concat(dataFixture),
+  };
+
+  handlePaginationChange = (newPagination) => {
+    const {rawData} = this.state;
+    const totalItems = rawData.length;
+    const {limit, page} = newPagination;
+    const totalPages = Math.ceil(totalItems / limit);
+    const sliceFrom = limit * page - limit;
+    const displayTo = page === totalPages ? totalItems : sliceFrom + limit;
+    const data = [].concat(rawData).slice(sliceFrom, displayTo);
+
+    this.setState({
+      data,
+      limit,
+      page,
+    });
+  };
+
+  render() {
+    const {data, limit, page} = this.state;
+    const {
+      entityNamePlural,
+      entityNameSingular,
+      totalItems,
+    } = this.props;
+
+    return (
+      <Pagination
+        entityNamePlural={entityNamePlural}
+        entityNameSingular={entityNameSingular}
+        isLoading={false}
+        limit={limit}
+        onPaginationChange={this.handlePaginationChange}
+        page={page}
+        totalItems={totalItems}
+      >
+        <ExternalTable data={data} />
+      </Pagination>
+    );
+  }
+
+}
+
+export default <Wrapper />;
+````
+
+## Running tests
+Run `yarn test` at the root of the monorepo
+
+
+## Running Cosmos
+Run `yarn cosmos` at the root of the monorepo
+Cosmos fixture is ready, but Cosmos setup is not. Expect an update once we configure Cosmos.
+
+<Props of={Pagination} />
+
+<Playground>
+  <Pagination
+    entityNamePlural={'Employees'}
+    entityNameSingular={'Employee'}
+    isLoading={false}
+    limit={10}
+    page={1}
+    count={100}
+  />
+</Playground>
```

Comment:
@lordpixel curious how you feel as a developer about what these two components generate and if you feel it is useful enough as is or not.